subprojects/platform-native/src/main/java/org/gradle/nativeplatform/tasks/PrefixHeaderFileGenerateTask.java

```diff
@@ -56,4 +73,20 @@ public File getPrefixHeaderFile() {
     public void setPrefixHeaderFile(File prefixHeaderFile) {
         this.prefixHeaderFile = prefixHeaderFile;
     }
+
+    private static class GeneratePrefixHeaderFile implements Runnable {
+        private final String header;
+        private final File prefixHeaderFile;
+
+        @Inject
+        public GeneratePrefixHeaderFile(String header, File prefixHeaderFile) {
+            this.header = header;
+            this.prefixHeaderFile = prefixHeaderFile;
+        }
+
+        @Override
+        public void run() {
+            PCHUtils.generatePCHFile(Lists.newArrayList(header), prefixHeaderFile);
```

Comment:
Maybe we should rename this "generatePrefixHeaderFile"?

Reply:
Done.